src/blog/lagom-1-3-8.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Lagom 1.3.8 is here!
+date: 2017-08-30
+author_github: ignasi35
+tags: news
+summary: >
+    Lagom 1.3.8 is ready
+---
+
+The Lagom team has released Lagom 1.3.8. For a list of the specific changes in Lagom 1.3.8, see the [change log](/changelog.html).
+
+This release fixes [a deadlock](https://github.com/lagom/lagom/issues/974) when shutting down a Lagom process in some scenarios and brings Scala DSL Json Serialization feature-parity by providing an opt-in [compressor](https://github.com/lagom/lagom/issues/955) for serialized messages. We also [upgraded](https://github.com/lagom/lagom/issues/968) the underlying logback library.
+
+This release is no different than previous ones and we continue to have great community contributions. Thanks to GitHub users [t-h-a-l-i-s](https://github.com/t-h-a-l-i-s) and [Jimin Hsieh](https://github.com/jiminhsieh).
+
+
+## Json Serialization compression
+
+Compression, as described here, is only used for persistent events, persistent snapshots and remote messages within the service cluster. It is not used for messages that are serialized in the external API of the service.
+
+JSON can be rather verbose and for large messages it can be beneficial to enable compression. That is done by using the `JsonSerializer.compressed[T]` builder method instead of the `JsonSerializer.apply[T]` (as shown in the example snippet above):
```

Comment:
the snippet is below, not above.

Reply:
Ha, then `current` docs are wrong. 😂 

Well, no, it's correct but the wording is confusing.

Reply:
https://www.lagomframework.com/documentation/1.4.x/scala/Serialization.html#Compression

Reply:
actually, in this context the sentence in parens is wrong. In the linked docs it's not wrong but confusing.